docs/source/user-guide/kubernetes.md

```diff
@@ -65,31 +65,31 @@ metadata:
 spec:
   type: Scala
   mode: cluster
-  image: ghcr.io/apache/datafusion-comet:spark-3.4-scala-2.12-0.5.0
+  image: apache/datafusion-comet:0.7.0-spark3.5.4-scala2.12-java11
```

Comment:
I have not actually tested the changes in this doc. @comphead does this look correct to you?

Reply:
I did test pulling the image, but not running this job

Reply:
it looks correct to me, image is pulled.
I'll check the job later and add the real world example